install-k6.js

```diff
@@ -0,0 +1,91 @@
+const { exec } = require('child_process')
+const { existsSync } = require('fs')
+
+const K6_VERSION = 'v0.55.0'
+const K6_PATH_MAC_AMD = `k6-${K6_VERSION}-macos-amd64`
+const K6_PATH_MAC_ARM = `k6-${K6_VERSION}-macos-arm64`
+const K6_PATH_WIN_AMD = `k6-${K6_VERSION}-windows-amd64`
+
+const existsK6 = (os, arch) => {
+  return existsSync(`resources/${os}/${arch}/k6`)
```

Comment:
This isn't working on Windows because of the missing extension (`.exe`) 😞 We may have to do the os check inside this function instead.

Reply:
We can remove this function now that we are using `existsSync` directly.